tuf/src/repository/ephemeral.rs

```diff
@@ -122,10 +121,104 @@ where
     }
 }
 
+/// [EphemeralTransaction] is a special repository that is designed to atomically commit metadata
```

Comment:
If this isn't truly atomic (i.e. it either all succeeds or all fails), I would stay away from 'atomic'. Perhaps instead use language like `commit metadata in a batch`?

Reply:
`EphemeralTransaction::commit()` is actually atomic, since it cannot fail (other than if something panics, but that would tear down the process). It's `FileSystemTransaction::commit()` that's not truly atomic.

Reply:
Even though it's atomic, I renamed it to EphemeralBatchUpdate to be consistent.